package.json

```diff
@@ -275,8 +275,6 @@
 		"test-unit-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist --verbose'",
 		"pretest-unit-php-multisite": "wp-env start",
 		"test-unit-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit/multisite.xml --verbose'",
-		"test-unit:native": "cd test/native/ && cross-env NODE_ENV=test jest --config ./jest.config.js",
-		"test-unit:native:debug": "cd test/native/ && node --inspect ../../node_modules/.bin/jest --runInBand --config ./jest.config.js",
```

Comment:
I saw that this command is referenced in the [testing documentation](https://github.com/WordPress/gutenberg/blob/rnmobile/try/unify-unit-test-cmd/docs/contributors/code/testing-overview.md#native-mobile-testing) so it would be nice if we update the command there too.